### PR TITLE
Fix incorrect import

### PR DESCRIPTION
**Pull request by ericvergnaud**
## Changes
Fix incorrect import

### Linked issues
None

### Functionality 

- [ ] added relevant user documentation
- [ ] added new CLI command
- [ ] modified existing command: `databricks labs ucx ...`
- [ ] added a new workflow
- [ ] modified existing workflow: `...`
- [ ] added a new table
- [ ] modified existing table: `...`

### Tests
- [ ] manually tested
- [ ] added unit tests
- [ ] added integration tests
- [ ] verified on staging environment (screenshot attached)

### CODE REVIEW

**Review by JCZuurmond — APPROVED** [review]
lgtm

File: src/databricks/labs/ucx/source_code/graph.py [hunk 1]
```diff
@@ -192,7 +192,9 @@ def _process_node(self, base_node: NodeBase):
             else:
                 yield from self.register_notebook(Path(strpath))
         if isinstance(base_node, ImportSource):
-            prefix = ("." * base_node.node.level) if isinstance(base_node.node, ast.ImportFrom) else ""
+            prefix = ""
+            if isinstance(base_node.node, ImportFrom) and base_node.node.level is not None:
+                prefix = "." * base_node.node.level
```
Comment (asnare) [inline src/databricks/labs/ucx/source_code/graph.py:197]:
```suggestion
            if isinstance(base_node.node, ImportFrom) and base_node.node.level is not None:
                prefix = "." * base_node.node.level
            else:
                prefix = ""
```

**Review by asnare — APPROVED** [review]

File: src/databricks/labs/ucx/source_code/graph.py (hunk 1 above)
Comment (nfx) [reply src/databricks/labs/ucx/source_code/graph.py:197]: @asnare That looks less readable

**Review by nfx — APPROVED** [review]
Lgtm

**Pull request merged**